frontend/src/pages/stakingOverview/Validators.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Typography, Table, TableContainer, TableHead, TableBody, TableRow, TableCell } from '@mui/material';
+import { Validator } from './Validator';
+
+export const Validators = (props) => {
+  return (
+    <>
+      <TableContainer>
+        <Table>
+          <TableHead>
+            <TableRow >
+              <TableCell>Validator Name</TableCell>
+              <TableCell>Staked Amount</TableCell>
+              <TableCell>Rewards</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {props.validators.map((validator) => (
+              <Validator validator={validator} key={validator.validatorName} denom={props.denom} />
```

Comment:
You should add a key to each list element [ref](https://legacy.reactjs.org/docs/lists-and-keys.html#keys).

Reply:
done